package transport;

enum msg_type {
  FORWARD_REQ = 0;
  FORWARD_MSG = 1;
};

message HolePunchingMsg {
  optional string ip = 1;
  optional int32 port = 2;
  optional msg_type type = 3;
}
